package.json

```diff
@@ -50,9 +50,9 @@
     "rimraf": "^2.5.2",
     "source-map-loader": "^0.1.5",
     "ts-loader": "^0.8.2",
-    "typescript": "^1.8.9",
+    "typescript": "^2.0.0",
```

Comment:
`npm i typescript@beta -D` is creating ` "typescript": "^2.0.0", `  in package.json. Is that what you want?